src/app/features/suggested-first-steps/suggested-first-steps.tsx

```diff
@@ -0,0 +1,54 @@
+import { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { openInNewTab } from '@app/common/utils/open-in-new-tab';
+import { useAppDispatch } from '@app/store';
+import { onboardingActions } from '@app/store/onboarding/onboarding.actions';
+import { useSuggestedFirstStepsStatus } from '@app/store/onboarding/onboarding.selectors';
+import {
+  SuggestedFirstSteps as Steps,
+  SuggestedFirstStepStatus,
+} from '@shared/models/onboarding-types';
+
+import { useSuggestedFirstSteps } from './hooks/use-suggested-first-steps';
+
+import { SuggestedFirstStepsLayout } from './suggested-first-steps.layout';
+
+export function SuggestedFirstSteps() {
+  const analytics = useAnalytics();
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+  const suggestedFirstStepsStatus = useSuggestedFirstStepsStatus();
+  useSuggestedFirstSteps();
+
+  const onDismissSteps = useCallback(() => {
+    void analytics.track('dismiss_suggested_first_steps');
+    dispatch(onboardingActions.hideSuggestedFirstSteps(true));
+  }, [analytics, dispatch]);
+
+  const onSelectStep = useCallback(
+    (route, event) => {
+      if (event === Steps.ExploreApps) {
+        dispatch(
+          onboardingActions.updateSuggestedFirstStepsStatus({
+            ...suggestedFirstStepsStatus,
+            [Steps.ExploreApps]: SuggestedFirstStepStatus.Done,
+          })
+        );
+      }
+      void analytics.track('select_next_step', { step: event });
+      if (route.startsWith('/')) navigate(route);
+      else openInNewTab(route);
+    },
+    [analytics, dispatch, navigate, suggestedFirstStepsStatus]
+  );
```

Comment:
Passing in content here, there's no need to have a conditional `onSelectStep`, each action handles it own specifc behaviour.